packages/mantine/src/components/prompt/Prompt.tsx

```diff
@@ -1,43 +1,121 @@
-import {Box, Modal, ModalProps} from '@mantine/core';
+import {
+    Box,
+    createVarsResolver,
+    factory,
+    Factory,
+    Image,
+    Modal,
+    ModalRootProps,
+    ModalStylesNames,
+    StylesApiProps,
+    useProps,
+    useStyles,
+} from '@mantine/core';
 import {Children, ReactElement, ReactNode} from 'react';
-import PromptClasses from './Prompt.module.css';
+import classes from '../header/Header.module.css';
```

Comment:
I think the import is wrong here

```suggestion
import classes from './Prompt.module.css';
```